vortex-datafusion/src/convert/exprs.rs

```diff
@@ -207,6 +207,8 @@ pub(crate) fn can_be_pushed_down(df_expr: &PhysicalExprRef, schema: &Schema) ->
         return false;
     }
 
+    let _expr_str = format!("{:?}", df_expr);
```

Comment:
I assume this is debugging? `dbg!` is useful in these cases

Reply:
This was leftover from debugging, but the rationale was that I wanted a string variable when I stepped thru this in the debugger.